apps/shooting-clubs/fields/index.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
@joefitter Once this is merged, there will be another PR with this file having stuff init!